.vib/rails/goss/rails.yaml

```diff
@@ -0,0 +1,12 @@
+command:
+  # A new Rails project can be created
+  check-project:
```

Comment:
Nice test, though I would include a server initialization here.

Reply:
I've added server initialization, it needs to be in the same test, as Goss tests aren't executed in the listed order.